src/main/java/com/arpnetworking/tsdcore/sinks/WavefrontSink.java

```diff
@@ -66,9 +67,25 @@ public void recordAggregateData(final PeriodicData periodicData) {
             // Build the list of annotations based on cluster and white listed dimensions
             final Map<String, String> annotations = Maps.newHashMap();
             annotations.put("cluster", fqdsn.getCluster());
```

Comment:
Hrm. I guess we haven't refactored it here yet. In MAD these fields are gone. This is fine.